components/retriever_eval_ragas/requirements.txt

```diff
@@ -0,0 +1,5 @@
+ragas==0.0.20
+transformers==4.35.2
+torch==2.1.1
+torchaudio==2.1.1
+torchvision==0.16.1
```

Comment:
Since we only use ragas explicitly, shouldn't these dependencies be installed by ragas?